cloudhub-2/modules/ROOT/pages/ch2-configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,138 @@
+= Configuring Horizontal Autoscaling (HPA) for CloudHub 2.0 Deployments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+[IMPORTANT]
+This feature applies only to select customers in the US control plane who are opted in to the new pricing and packaging model. For more details, visit xref:general::pricing.adoc[]
```

Comment:
On the Note: Why specify US control plane -- can we just say for customers opted into the new P&P? This feature applies only to select customers who are opted in to the new pricing and packaging model. For more details, visit general::pricing.adoc